tests/Unit/Listener/DidChangeWatchedFilesListenerTest.php

```diff
@@ -27,9 +29,11 @@ class DidChangeWatchedFilesListenerTest extends TestCase
 
     public function testDynamicallyRegisterIfSupported(): void
     {
-        $this->initListener(new ClientCapabilities([
-            'didChangeWatchedFiles' => ['dynamicRegistration' => true],
-        ]));
+        $this->initListener(new ClientCapabilities(
+            workspace: new WorkspaceClientCapabilities(
+                didChangeWatchedFiles: new DidChangeWatchedFilesClientCapabilities(dynamicRegistration: true),
+            )
```

Comment:
looks like this is a BC break in the protocol, maybe more like this in Phpactor.

Reply:
... or maybe this was just plain wrong.